index.html

```diff
@@ -291,6 +303,63 @@
           </dt>
         </dl>
       </section>
+      <section>
+	<h2>
+	  Media Devices
+	</h2>
+	<p>
+	  The <code><dfn id='idl-def-PermissionName.camera'>camera</dfn></code>,
+	  <code><dfn id='idl-def-PermissionName.microphone'>microphone</dfn>
+	  </code>,
+	  and <code><dfn id='idl-def-PermissionName.speaker'>speaker</dfn>
+	  </code>
+	  permissions are associated with permission to use media
+	  devices as specified in [[GETUSERMEDIA]] and
+	  [[audio-output]].
+	</p>
+	<p>These three permissions have an associated PermissionDescriptor:
+	</p>
+	<dl class='idl' title='dictionary DevicePermissionDescriptor :
+	    PermissionDescriptor'>
+	  <dt>
+	    DOMString deviceId
+	  </dt>
+	</dl>
+	<p>A permission covers access to the device given in the
+	associated descriptor.
+	</p>
+	<p>
+	  If the descriptor does not have a deviceId, its semantic is that
+	  it queries for access to any device of that class.
+	</p>
+	<p>
+	  If the deviceId is the special string "*", it queries for
+	  access to all devices of that class, whether they exist
+	  at the moment or are attached at a later time. When such a
```

Comment:
Please suggest text... for instance, we could use the strings "some device" for <empty> and "all devices" for "*" - this revisits the question of what values of a device ID can be used safely for something other than a device id - or we could add another attribute.
My first proposal (separate "all-" permissions) was a bit easier to explain, but did not support the "any" case.